saleor/graphql/checkout/mutations.py

```diff
@@ -628,6 +628,13 @@ def perform_mutation(cls, _root, info, checkout_id, shipping_method_id):
 
 class CheckoutComplete(BaseMutation):
     order = graphene.Field(Order, description="Placed order.")
+    confirmation_needed = graphene.Boolean(
```

Comment:
I assume that after we added this flag we should also modify the description of the `class Meta` as there is a possibility that we won't create new order.